fabric/build.gradle

```diff
@@ -105,3 +105,8 @@ sourcesJar {
     dependsOn commonSources
     from commonSources.archiveFile.map { zipTree(it) }
 }
+
+spotless.json {
+    // schemaVersion must be at the top or else Fabric will complain
+    targetExclude("src/main/resources/fabric.mod.json")
+}
```

Comment:
I don't actually like this. Not sure if we requested this, or you thought it's a good idea, but I value all spotless rules being in a single place. Please revert it.

Reply:
Sorry, I added it here since I couldn't get it to work in the main build.gradle file but I figured it out and I moved it.